service/history/historyEngineInterfaces.go

```diff
@@ -106,15 +106,23 @@ type (
 
 	timerQueueProcessor interface {
 		common.Daemon
-		NotifyNewTimers(timerTask []persistence.Task)
+		NotifyNewTimers(clusterName string, timerTask []persistence.Task)
+		SetCurrentTime(clusterName string, currentTime time.Time)
+	}
+
+	timerProcessor interface {
```

Comment:
Naming is quite confusing between timerQueueProcessor and timerProcessor.

Reply:
any suggestion?